rhel8/profiles/e8.profile

```diff
@@ -123,14 +123,16 @@ selections:
   - sshd_print_last_log
   - sshd_use_priv_separation
   - sshd_do_not_permit_user_env
-  - sshd_disable_rhosts_rsa
   - sshd_disable_rhosts
-  - sshd_allow_only_protocol2
   - sshd_set_loglevel_info
   - sshd_disable_empty_passwords
   - sshd_disable_user_known_hosts
   - sshd_enable_strictmodes
 
+  - var_system_crypto_policy=default
```

Comment:
@shaneboulden is there a crypto policy requirement for Essential 8 like FIPS is for US Gov?

Reply:
@redhatrises There's nothing explicit for the Essential 8, though we'd try to follow the [ISM guidelines for using crypto](https://www.cyber.gov.au/ism/guidelines-using-cryptography), and specifically the 'ASD Approved Crypto Algorithms'

It looks like the `default` policy meets a lot of these requirements with the exception that it permits SHA-1 in digital signatures (ASD approves SHA-2 family).

I think we'd probably prefer to use a `future` system-wide policy. The guidelines provided (linked above) are generally conservative, and this meets the requirement for SHA-2 (and explicitly disallowing SHA-1).